spec/features/admin/admin_invoice_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.describe "Admin Invoice Page", type: :feature do
```

Comment:
It looks like tests for the **Admin Invoice Show** page and the **Admin Invoice Index** page are combined into one file.

Can you separate these and make each page have its own spec file?

Reply:
The show page tests are now in spec/admin/admin_show_spec